.travis.yml

```diff
@@ -44,14 +44,15 @@ matrix:
     env: COMPILER=clang++-6.0 QT=512 DISABLE_IPC=OFF MONOLITHIC_BUILD=OFF
 
 install:
-- sudo apt-get install python3 python3-pip python3-setuptools doxygen libgl1-mesa-dev
+- sudo apt-get install python3 python3-pip python3-setuptools graphviz doxygen libgl1-mesa-dev
 - sudo pip3 install antlr4-python3-runtime six pyyaml click typing jinja2 watchdog path.py
 
 before_script:
 - QTDIR="/opt/qt${QT}"
 - PATH="$QTDIR/bin:$PATH"
 - source /opt/qt${QT}/bin/qt${QT}-env.sh
 - cd ${TRAVIS_BUILD_DIR}
+- source setup_cmake.ch
```

Comment:
for consistency, please do not use an external script.

Reply:
I did this like in Gtest installation bellow:
 - source setup_gtest.sh
Should I move the text setup_cmake.sh to .travis.yml now?

Reply:
Using external script makes sense to me. They are going to update CMake on Travis sooner or later. Removing one line looks better. I would not use "consistency" argument between two so different environments.
 
GTest is a bit different story though. This step is not done on Windows.

Reply:
If you want to use an external script, please do it consistently (for both linux and windows builds) and move the scripts into a folder ("CI" ?) so that our root folder does not get overpopulated.

Reply:
Done